orderbook/src/main.rs

```diff
@@ -127,6 +127,14 @@ struct Arguments {
         parse(try_from_str = shared::arguments::duration_from_seconds),
     )]
     solvable_orders_max_update_age: Duration,
+
+    #[structopt(
```

Comment:
Can you include a doc comment here? It gets used to automatically generate the documentation in `--help`.